lookup.go

```diff
@@ -34,17 +35,21 @@ type routineMetadata struct {
 }
 
 func GetDNSServers(path string) ([]string, error) {
-	c, err := dns.ClientConfigFromFile(path)
-	if err != nil {
-		return []string{}, err
-	}
 	var servers []string
-	for _, s := range c.Servers {
-		if s[0:1] != "[" && strings.Contains(s, ":") {
-			s = "[" + s + "]"
+	if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
```

Comment:
I don't think that it makes sense for defaults to be specified deep in a function like this. Rather, they should be specified as default values elsewhere, e.g., https://github.com/zmap/zdns/blob/master/zdns/main.go. It also likely makes more sense for them to be added in if the call to `GetDNSServers` fails, rather than being specified in the call itself. Can you investigate trying to move this check upwards in the ZDNS stack?

Reply:
Ok

Reply:
complete

Reply:
@zakird Did I do wrong?